zk-tests/src/Cheatcodes.t.sol

```diff
@@ -155,16 +155,17 @@ contract ZkCheatcodesTest is Test {
         assertEq(dataAfter, bytes(hex"a1b1"));
     }
 
-     function testZkCheatcodesCanMockCallTestContract() public {
-        address thisAddress = address(this);
+    function testZkCheatcodesCanMockCall(address mockMe) public {
```

Comment:
We should keep this test as is, and create a new fuzz test, since this test solves a specific case that's not guaranteed to be in a fuzz input